Makefile

```diff
@@ -23,7 +23,13 @@ run-mock-registry: ${GOPATH}/bin/mock-registry vendor
 prepare-build: install
 	cp "${GOPATH}"/bin/broker build/broker
 
-build: prepare-build
+dev-image: prepare-build
+	docker build -f ${BUILD_DIR}/Dockerfile-src ${BUILD_DIR} -t ${BROKER_IMAGE}:${TAG}
+	@echo
+	@echo "Remember you need to push your image before calling make deploy"
+	@echo "    docker push ${BROKER_IMAGE}:${TAG}"
+
+release-image:
```

Comment:
Talked to @jmrodi. Think we're going to do
make release-image
make push
make release: release-image

We may change `make release` to be `make release: release-image push` later but for now we'll leave them as discrete tasks.

Sounds like `make build` might get used for a go build and `make install` changed to install the broker in /usr/local/bin or something so it might be a good time to free up `make build` in case we make that change.